scripts/deploy_ai.sh

```diff
@@ -71,4 +71,22 @@ gcloud beta ai endpoints deploy-model "${ENDPOINT_ID}" \
     --model="${MODEL_ID}" \
     --display-name="${MODEL_NAME}" \
     --machine-type="${MACHINE_TYPE}" \
-    --accelerator="count=1,type=nvidia-tesla-t4"
\ No newline at end of file
+    --accelerator="count=1,type=nvidia-tesla-t4"
+
+# Extract model id from an URL like:  http://ENDPOINT_ID.aiplatform.googleapis.com/v1/models/DEPLOYED_MODEL_ID:predict
+DEPLOYED_MODEL_ID=$(gcloud beta ai endpoints describe ${ENDPOINT_ID} --project="${PROJECT_ID}" --region="${REGION}" --format="value(deployedModels.privateEndpoints.predictHttpUri)" | sed -e 's|.*models/\(.*\):predict|\1|g')
+
+ENDPOINT_ID=123
+DEPLOYED_MODEL_ID=1234
```

Comment:
remove 2 lines? Also add some comments on what this print helps?

Reply:
Yeah thanks :)